pkg/bus/monitor.go

```diff
@@ -11,61 +10,10 @@ import (
 	"go.uber.org/fx"
 )
 
-const (
```

Comment:
Why move all those events to core? There are never used in this package.

Reply:
you're right, moved back!